pkg/storaged/block/format-dialog.jsx

```diff
@@ -290,7 +290,7 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
     let [, old_dir, old_opts] = get_fstab_config(block, true,
                                                  content_block?.IdType == "btrfs"
                                                      ? { pathname: "/", id: 5 }
-                                                     : null);
+                                                     : undefined);
```

Comment:
Does that have any observable effect?

Reply:
Yes, get_fstab_config explicitly checks for `undefined` and would try to access `null.pathname`, for example. 

This was a bug in [storage: Improve Format dialog defaults](https://github.com/cockpit-project/cockpit/pull/19984/commits/721e1d6fe2cb7118b78e50b67d12d8dcd6bee45a), and I put it into the wrong commit... DO we care?

Reply:
Ah, it's already merged. :-)